addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -515,6 +515,11 @@ var FieldMany2One = AbstractField.extend({
     _renderEdit: function () {
         var value = this.m2o_value;
 
+        const escapedValue = _.escape(`${value}`.trim());
+        const [, ...lines] = escapedValue.split('\n');
+        const extraText = lines.map((line) => `<span>${line}</span>`).join('<br/>');
+        this.$('.o_field_many2one_extra').html(extraText);
```

Comment:
same as in readonly? factorize the code then. Note that building html in JS isn't what we like to see (xss risks)